### PR TITLE
Javascript/javascript3/week1

### PR DESCRIPTION
**Pull request by arya56**

### CODE REVIEW

**Comment by bashirmja** [comment]
Hi Arya, 
I am going to review your code but I saw that you just included all files from other weeks and therefore I think you should clean up your branch. 
So can you please delete all files from your branch which is not related to this homework?
Thanks 
Bashir

File: javascript/javascript3/week1/weather.html [hunk 1]
```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <!-- <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.1.0/css/all.css"> -->
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css" rel="stylesheet">
+    <!-- <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"> -->
```
Comment (bashirmja) [inline javascript/javascript3/week1/weather.html:6]: Please remove the commented code in lines 4  and 6.

File: javascript/javascript3/week1/style.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#center , .center2 {
+    margin: auto;
+    width: 60%;
+    border: 3px solid #73AD21;
+    padding: 70px;
+
+}
+/* .center2 {
```
Comment (bashirmja) [inline javascript/javascript3/week1/style.css:8]: Also, remove commented css code here and at line 24.

File: javascript/javascript3/week1/cityWeather.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+                // Display the data
+                // This data should be showed in your app:
+
+                // The chosen city
+                // Temperature
+                // Icon for the weather type
+                // Wind speed
+                // How clowdy it is
+                // When sunrise and sunset is
+                // Optional a map showing where the city is located
```
Comment (bashirmja) [inline javascript/javascript3/week1/cityWeather.js:11]: I am not sure if we need also these comments.

**Review by bashirmja — CHANGES_REQUESTED** [review]
Besides those small commented codes, the rest looks good. As mentioned please remove other files not related to this week's homework.

**Comment by arya56** [comment]
I applied all reviews!

**Pull request merged**